<ui version="4.0" >
 <class>UiNewEditUserBase</class>
 <widget class="QWidget" name="UiNewEditUserBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>399</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>644</width>
    <height>399</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>644</width>
    <height>399</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>New / Edit User</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../resource/dms.qrc" >
    <normaloff>:/picture/32/images/32x32/user_32.png</normaloff>:/picture/32/images/32x32/user_32.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QTreeWidget" name="treeWidgetUser" >
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Username</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Userpassword</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Firstname</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Lastname</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>User ID:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditUserId</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUserId" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="frame" >
        <bool>true</bool>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUserId" >
       <property name="minimumSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../resource/dms.qrc" >
         <normaloff>:/picture/16/images/16x16/reload_16.png</normaloff>:/picture/16/images/16x16/reload_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelFirstName" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Firstname:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditUserFirstName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUserFirstName" >
       <property name="frame" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item rowspan="4" row="2" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QPushButton" name="btnAddUser" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpdateUser" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteUser" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelLastName" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Lastname:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditUserLastName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUserLastName" >
       <property name="frame" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Username:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditUserName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUserName" >
       <property name="frame" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Password:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditUserPassword</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUserPassword" >
       <property name="frame" >
        <bool>true</bool>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Confirm password:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditUserPasswordConfirm</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUserPasswordConfirm" >
       <property name="frame" >
        <bool>true</bool>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeWidgetUser</tabstop>
  <tabstop>lineEditUserId</tabstop>
  <tabstop>lineEditUserFirstName</tabstop>
  <tabstop>lineEditUserLastName</tabstop>
  <tabstop>lineEditUserName</tabstop>
  <tabstop>lineEditUserPassword</tabstop>
  <tabstop>lineEditUserPasswordConfirm</tabstop>
  <tabstop>btnAddUser</tabstop>
  <tabstop>btnUpdateUser</tabstop>
  <tabstop>btnDeleteUser</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../resource/dms.qrc" />
 </resources>
 <connections/>
</ui>
